app/build.gradle

```diff
@@ -194,6 +194,14 @@ android {
         }
     }
 
+    applicationVariants.configureEach { variant ->
+        variant.outputs.each { output ->
+            if (variant.buildType.name == "debug") {
+                output.versionCodeOverride = 999
```

Comment:
We should use a much larger versionCode for local builds to ensure that we can always install it over published packages. Check the version generator code and you'll see which kind of version codes it generates.

Reply:
Good point, I set it to a value that refers to some date in the year 2115. I also created the configuration property that you suggested in the above comment. I couldn't find `user.properties` so I added it to gradle.properties, I hope that's ok.